src/main/protobuf/worker_protocol.proto

```diff
@@ -38,6 +38,8 @@ message WorkRequest {
   // The inputs that the worker is allowed to read during execution of this
   // request.
   repeated Input inputs = 2;
+
+  int32 request_id = 3;
```

Comment:
What kind of ID is this? Why is it necessary? Please document.

Reply:
For worker process to run in parallel, we need a way to distinguish which request corresponds to which worker. Docstrings added.